utils/das_data_sorter.py

```diff
@@ -136,6 +137,53 @@ def gps_data(self, data:pd.Series) -> pd.Series:
         
         return new_gps_data
     
+    def smooths_data(self, data:pd.Series, technique:str, n:int=3) -> pd.Series:
```

Comment:
This function is quite large with lots of if statements, which can be difficult to maintain - I'd probably make two smaller functions, one to handle mean and one to handle median, which this function may call.